### ISSUE TITLE
[renamerOnUpdate] Studio Rename Not Working Properly

### ISSUE THREAD

**Issue by Trux** [stashapp/CommunityScripts#114]
Stash v0.18.0
Plugin: renamerOnUpdate (2.3.5)

Expected Behavior:
When saving changes on a file, the file renames.

Actual Behavior:
Nothing happens.

Log Output:
[Plugin / Update_Hook] No template for this file.

Config.py 
```
#               TEMPLATE             #

# Priority : Tags > Studios > Default

# Templates to use for given tags
# Add or remove as needed or leave it empty/comment out
tag_templates = "$title - $performer"

# Adjust the below if you want to use studio names instead of tags for the renaming templates
studio_templates =  "[$studio]  $title - $performer"

# Change to True to use the default template if no specific tag/studio is found
use_default_template = True
# Default template, adjust as needed
default_template =  "$performer - $title_$height"  

######################################
#               Logging              #

# File to save what is renamed, can be useful if you need to revert changes.
# Will look like: IDSCENE|OLD_PATH|NEW_PATH
# Leave Blank ("") or use None if you don't want to use a log file, or a working path like: C:\Users\USERNAME\.stash\plugins\Hooks\rename_log.txt
log_file = r"./renamerLog.txt"

######################################
#               Settings             #

# Character which replaces every space in the filename
# Common values are "." and "_"
# e. g.:
# "."
# 2016-12-29.Eva.Lovia.-.Her.Fantasy.Ball
filename_splitchar = " "

# Character to use as a performer separator.
performer_splitchar = " "
# Maximum number of performer names in the filename. If there are more than that in a scene the filename will not include any performer name!
performer_limit = 3
# Ignore male performers.
performer_ignore_male = True

# If $performer is before $title, prevent having duplicate text. 
# e.g.:
# Template used: $year $performer - $title
# 2016 Dani Daniels - Dani Daniels in ***.mp4 --> 2016 Dani Daniels in ***.mp4
prevent_title_performer = True

# Squeeze studio names removes all spaces in studio, parent studio and studio family name
# e. g.:
# Reality Kings --> RealityKings
# Team Skeet Extras --> TeamSkeetExtras
squeeze_studio_names = True

# Rating indicator option to identify the number correctly in your OS file search
# Separated from the template handling above to avoid having only "RTG" in the filename for scenes without ratings
# e. g.:
# "{}" with scene rating of 5       == 5
# "RTG{}" with scene rating of 5    == RTG5
# "{}-stars" with scene rating 3    == 3-stars
rating_format = "{}"

# Character to use as a tag separator.
tags_splitchar = " "
# Include and exclude tags
# 	Tags will be compared strictly. "pantyhose" != "Pantyhose" and "panty hose" != "pantyhose"
# Option 1: If you're using whitelist, every other tag which is not listed there will be ignored in the filename
# Option 2: All tags in the tags_blacklist array will be ignored in the filename. Every other tag will be used.
# Option 3: Leave both arrays empty if you're looking for every tag which is linked to the scene. 
# 			Attention: Only recommended if the scene linked tags number is not that big due to maxiumum filename length
tags_whitelist = [
    # "Brunette", "Blowjob"
]

tags_blacklist = [
	# ignored tags...
]

# Only rename 'Organized' scenes.
only_organized = False

# If the new path is over 240 characters, the plugin will try to reduce it. Set to True to ignore that.
ignore_path_length = False
# Field to remove if the path is too long. First in list will be removed then second then ... if length is still too long.

order_field = ["$video_codec", "$audio_codec", "$resolution", "tags", "rating", "$height", "$studio_family", "$studio", "$parent_studio", "$performer"]

# Alternate way to show diff. Not useful at all.
alt_diff_display = False

######################################
#            Module Related          #

# ! OPTIONAL module settings. Not needed for basic operation !

# = psutil module (https://pypi.org/project/psutil/) =
# Gets a list of all processes instead of stopping after the first one. Enabling it slows down the plugin
process_getall = False
# If the file is used by a process, the plugin will kill it. IT CAN MAKE STASH CRASH TOO. 
process_kill_attach = False
# =========================

# = Unidecode module (https://pypi.org/project/Unidecode/) =
# Check site mentioned for more details. 
# TL;DR: Prevent having non common characters by replacing them.
# Warning: If you have non-latin characters (Cyrillic, Kanji, Arabic, ...), the result will be extremely different.
use_ascii = False 
# =========================

```

**Comment by Cycor** [comment]
You are using the templates wrong, it should be a dictionary

like this:
tag_templates  = {
	"rename_tag": "$year $title - $studio $resolution $video_codec",
	"rename_tag2": "$title"
}

studio_templates  = {
	"Blender Institute": "$date - $title [$studio]",
	"Pixar": "$title [$studio]"
}

rename_tag, rename_tag2 are the tags of the scene you want to rename.

examples are taken from the [README.md](https://github.com/stashapp/CommunityScripts/tree/main/plugins/renamerOnUpdate#--based-on-a-tag)

**Comment by Trux** [comment]
I tried but get the same error whether I use the example template or even leave it empty/comment out
```
####################################################################

#               TEMPLATE             #

# Priority : Tags > Studios > Default

# Templates to use for given tags
# Add or remove as needed or leave it empty/comment out
#tag_templates = {}

# Adjust the below if you want to use studio names instead of tags for the renaming templates
#studio_templates = {}

# Change to True to use the default template if no specific tag/studio is found
use_default_template = True
# Default template, adjust as needed
default_template =  "$date $title" 

```

**Comment by Cycor** [comment]
try starting with a clean config file, your file has a lot of differences to the [default file](https://github.com/stashapp/CommunityScripts/blob/main/plugins/renamerOnUpdate/renamerOnUpdate_config.py)
